CNAME

```diff
@@ -1 +0,0 @@
-read.wiki.org
```

Comment:
We may have to reconsider removing CNAME if this particular github repo isn't served properly once deployed. After completing several cycles through the github pages documentation I am still at a loss as to how routing takes place on their end. We must note that small-client-wiki is the second source of github pages from the WardCunningham account. My dns entry only gets to the front door.

- https://wardcunningham.github.io/
- https://wardcunningham.github.io/small-client-wiki

This second address is redirected to http://read.wiki.org by some magic that I assume comes from the CNAME file here. I assume the popularity of github pages has allowed a bit of a monster to grow.